clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,15 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateTOptions{mBlockSizeBytes=" + mBlockSizeBytes + ",mRecursive=" + mRecursive
```

Comment:
How about using Guava's toStringHelper for this?
http://docs.guava-libraries.googlecode.com/git-history/release/javadoc/com/google/common/base/MoreObjects.html#toStringHelper(java.lang.Object)